roles.json

```
@@ -44,7 +44,7 @@
       "url" : "Learn_coordinator",
       "sub-role" : [],
       "lead" : ["Kelle Cruz"],
-      "deputy" : ["Unfilled"],
+      "deputy" : ["Adrian Price-Whelan"],
```

After looking at this again it occurs to me that one could argue that this isn't really supposed to have a deputy, in that the "sub-roles" are really the deputies.  That is, there aren't "deputy" astropy coordinators because the other roles are sort of deputies.

I don't have a strong opinion, and am happy to defer to @kelle if she wants a specific deputy (and @adrn is certainly great for the role). But perhaps we should have the discussion in a separate PR?

```suggestion
      "deputy" : [],
```

---

Sounds reasonable to me!